.github/workflows/elixir.yaml

```diff
@@ -1,85 +0,0 @@
-# Adapted from: https://fly.io/phoenix-files/github-actions-for-elixir-ci/
-name: Elixir CI
-
-# Defines when the workflow runs
-on:
-  push:
-    branches: ["main"]
-  pull_request:
-    branches: ["main"]
-
-# Sets the ENV `MIX_ENV` to `test` for running tests
-env:
-  MIX_ENV: test
-
-permissions:
-  contents: read
-
-jobs:
-  test:
-    runs-on: ubuntu-latest
-    name: Test on OTP ${{matrix.otp}} / Elixir ${{matrix.elixir}}
-    strategy:
-      # Specify the OTP and Elixir versions to use when building
-      # and running the workflow steps.
-      matrix:
-        otp: ["25.0.4"]
-        elixir: ["1.14.1"]
-    steps:
-      # Step: Setup Elixir + Erlang image as the base.
-      - name: Set up Elixir
-        uses: erlef/setup-beam@v1
-        with:
-          otp-version: ${{matrix.otp}}
-          elixir-version: ${{matrix.elixir}}
-
-      # Step: Check out the code.
-      - name: Checkout code
-        uses: actions/checkout@v3
-
-      # Step: Define how to cache deps. Restores existing cache if present.
-      - name: Cache deps
-        id: cache-deps
-        uses: actions/cache@v3
-        env:
-          cache-name: cache-elixir-deps
-        with:
-          path: deps
-          key: ${{ runner.os }}-mix-${{ env.cache-name }}-${{ hashFiles('**/mix.lock') }}
-          restore-keys: |
-            ${{ runner.os }}-mix-${{ env.cache-name }}-
-
-      # Step: Define how to cache the `_build` directory. After the first run,
-      # this speeds up tests runs a lot. This includes not re-compiling our
-      # project's downloaded deps every run.
```

Comment:
How come we've removed the `_build` caching?

Reply:
Caching the `_build` folder should be handled by the setup-elixir-project action:

https://github.com/CargoSense/setup-elixir-project/blob/main/action.yml#L71-L77

Reply:
If it doesn't, that's a bug to be sorted out.

Reply:
OH! I misunderstood how we were using `setup-elixir-project`. Ignore me :)

Reply:
It's okay! The reusable action does _a lot_ so I had to look that up to verify.